### ISSUE TITLE
Add the backbone for tacocat to accept a generic set of geometries

### ISSUE THREAD

**Issue by lcarasik** [fastresearchgroup/TACOCAT#3]

**Comment by sierratutwiler** [comment]
Add fuel geometries as well as core geometries

**Comment by lcarasik** [comment]
This issue has been partially addressed in #38.

A few items need to be addressed or separated out as other issues and I'll update this as I have time.

This section towards the end of tacocat.py assumes all hexagonal bundles have wire wraps and needs to be generalized for with and without wire-wrap.

```
if Geometry_Type == "Hexagonal":
	M = ((1.034/(PtoD**0.124))+(29.7*(PtoD**6.94)*Re1**0.086)/(Geom.LeadW(FoCD,WoD)/FoCD)**2.239)**0.885 #modifier
else:
	M = 1
fsm = 0.316*Re1**(-0.25)
dP = M*fsm*(Hc/Core_Geometry[Geometry_Type]["InnerHydraulicDiameter"])*0.5*rhoavg*Uavg**2
```

**Comment by lcarasik** [comment]
@FranklinTrevor Please add a comment outlining what we talked about.

**Comment by FranklinTrevor** [comment]
Remove all hard coded geometry issues out of TACOCAT and into generic geometry folder

**Comment by FranklinTrevor** [comment]
This issue is resolved in pull request #47 @lcarasik 

TACOCAT has geometry dictionary file to choose from square, hexagonal, and wire wrapped hexagonal fuel bundles.
